.circleci/config.yml

```diff
@@ -69,6 +69,20 @@ jobs:
       - run:
           name: Check black code style
           command: black --check .
+  djlint:
```

Comment:
Ah wait, this job is not contained in the dev workflow, needs to be added there as well.

Reply:
This should be fixed now :)